lib/apiservers/engine/backends/container_proxy_test.go

```diff
@@ -42,27 +43,7 @@ func TestProcessVolumeParams(t *testing.T) {
 	assert.Equal(t, "testVolume", processedTestVolumes[2].ID)
 	assert.Equal(t, "/mount/path", processedTestVolumes[2].Dest)
 	assert.Equal(t, "r", processedTestVolumes[2].Flags)
-}
-
-func TestProcessSpecifiedVolumes(t *testing.T) {
-	rawTestVolumes := []string{"masterVolume:/blah", "testVolume:/mount:r", "specVol:/mount/path:r"}
-	var processedTestVolumes []volumeFields
-
-	processedFields, err := processSpecifiedVolumes(rawTestVolumes)
-	assert.Nil(t, err)
-	processedTestVolumes = append(processedTestVolumes, processedFields...)
 
```

Comment:
The new func, `processVolumeFields`, simply calls `processVolumeParam` on a slice. There's already a test for the latter, and I added a new test case where an error is expected.